library/ssl_cli.c

```diff
@@ -918,8 +918,8 @@ static int ssl_write_client_hello( mbedtls_ssl_context *ssl )
                                       ssl->conf->max_minor_ver ) != 0 )
             continue;
 
-        MBEDTLS_SSL_DEBUG_MSG( 3, ( "client hello, add ciphersuite: %04x",
-                                    ciphersuites[i] ) );
+        MBEDTLS_SSL_DEBUG_MSG( 3, ( "client hello, add ciphersuite: %04x (%s)",
```

Comment:
Since we print hexadecimal values, maybe it would be better to prefix them with `0x`, so that we don't lead to ambiguity, when e.g. "42" is printed.

Reply:
Yes, that's worth fixing along the way, too :+1:

Reply:
Fixed.